### ISSUE TITLE
Internal problem while using Kite

### ISSUE THREAD

**Issue by jmiszczak** [spyder-ide/spyder#12074]
### What steps will reproduce the problem?

I change the name of a variable (a list) and a Kite popup is opend with suggeted code.

### Traceback
```python-traceback
  File "/home/jam/Local/anaconda3/envs/puma/lib/python3.7/site-packages/spyder/plugins/completion/fallback/actor.py", line 155, in handle_msg
    msg['current_word'])
  File "/home/jam/Local/anaconda3/envs/puma/lib/python3.7/site-packages/spyder/plugins/completion/fallback/actor.py", line 62, in tokenize
    valid = is_prefix_valid(text, offset, language)
  File "/home/jam/Local/anaconda3/envs/puma/lib/python3.7/site-packages/spyder/plugins/completion/fallback/utils.py", line 122, in is_prefix_valid
    current_pos_text = text[offset - 1]
IndexError: string index out of range
```

## Versions

* Spyder version: 4.1.1 
* Python version: 3.7.6
* Qt version: 5.12.5
* PyQt5 version: 5.12.3
* Operating System: Linux 4.4.131-0404131-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
keyring                        :  None (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.2 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  2.4.4 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
xdg >=0.26                     :  0.26 (OK)
zmq >=17                       :  19.0.0 (OK)

# Optional:
cython >=0.21                  :  0.29.16 (OK)
matplotlib >=2.0.0             :  3.2.1 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by ccordoba12** [comment]
Duplicate of #11862

**Comment by ccordoba12** [comment]
It'll be fixed in our next version (4.1.2), to be released in the next couple of days.
